Suggestion #1091 - Completed

CayeOficial opened the pull request:
This PR corresponds to the Suggestion #1091 which has been accepted and completed.
After some research, I found out that PostgreSQL is not really as used as urgent to put in a guide, and it's most likely not needed because it's not a common Database type required for games, webapps and similar.

Also, I have had problems because when I wanted to write the MongoDB guide for Ubuntu I would have had to create a subcategory of a subcategory of a subcategory. So in order not to have 3 subcategories, I have opted to do this:

https://cdn.cayeoficial.com/image/opera_7fm9lEMRuJ.png

I don't know if it's the best way to avoid creating another tab list for ubuntu 22.04, 20.04...

ThatGuyJacobee commented:
Hey again Caye 👋

I've read over your PR and I have done a few tweaks and changes to make the guide a bit more in line with our guidelines. I was generally very satisfied with the guide, it was informative and well made so great job! 😄

One small point that you should remember for the future is to try to avoid using 'we' when referencing what to do. For example, rather than saying "we will now use the following command to do XYZ", it is best to use "you will now use the following command to do XYZ" as we want our guides to keep the reader engaged and keep the focus on them.

But otherwise, I liked your use of the tabs for different Linux distros and different database systems and it was overall well made. Good job! 

As always, we now need to perform our final checks and translations and we will be happy to publish the guide and provide you with your reward very soon. Once again, sorry for the longer delay than usual as we have been very busy. 👌

CayeOficial commented:
Hello,
I apologize for any errors or imbalance that my guide may have had. I will take the advice into account for next time.

The last two commits I have created make the mariadb installation guide on ubuntu/debian simpler, for the following reasons:

- I have removed the step of creating the file containing the MariaDB repo source, I had put it just in case creating it with `tee` failed because of permissions, but as it is executed with sudo there should be no problem.

- I removed the operating system alias table, and replaced it with the following code: `$(grep VERSION_CODENAME /etc/os-release | awk -F= '{print $2}')`.
When the user executes the command, the system alias (or codename) will be obtained directly, avoiding additional burden for the user.

Again, thanks for the opportunity to participate in the docs.

CayeOficial commented:
I was checking and realized that you made a mistake when you put together the mariadb installation guides for ubuntu and debian into one. The repo used for ubuntu is `https://repo.mariadb.org/apt/ubuntu`, and for debian `https://repo.mariadb.org/apt/debian`. However you have put together the debian one with the ubuntu command.

My latest commit should fix that error, as it uses the following command `$(grep "^ID=" /etc/os-release | awk -F= '{print $2}')` which will return "ubuntu" or "debian" dynamically.
